macros/cross_db_utils/dateadd.sql

```diff
@@ -16,8 +16,8 @@
 
 {% macro bigquery__dateadd(datepart, interval, from_date_or_timestamp) %}
 
-        datetime_add(
-            cast( {{ from_date_or_timestamp }} as datetime),
+        timestamp_add(
+            cast( {{ from_date_or_timestamp }} as timestamp),
```

Comment:
I'm not sure why we preferred `datetime` here. Looks like this change was made a long time ago (https://github.com/fishtown-analytics/dbt-utils/commit/85a615f2a6962febc8804b9aa7a9ce0d9945f38b).

<img width="561" alt="Screen Shot 2021-01-06 at 12 51 49 PM" src="https://user-images.githubusercontent.com/13897643/103765888-339b7880-501e-11eb-94ad-5bdc49172c88.png">

Agree that this is a _potentially_ breaking change—I think one worth making—and that we should make it in `datediff` too:
https://github.com/fishtown-analytics/dbt-utils/blob/ac78d49590ca04758d8ecdeb042eb8152bba6b0f/macros/cross_db_utils/datediff.sql#L17-L25